int8 robotTalking
int8 robotBouncing